internal/experimental/experimental.go

```diff
@@ -0,0 +1,75 @@
+package experimental
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	envName = "NOTATION_EXPERIMENTAL"
+	enabled = "1"
+)
+
+// IsDisabled determines whether an experimental feature is disabled.
+func IsDisabled() bool {
+	return os.Getenv(envName) != enabled
+}
+
+// Error returns an error for a disabled experimental feature.
+func Error(description string) error {
+	return fmt.Errorf("%s been marked as experimental and not enabled by default. To use it, please set %s=%s in your environment", description, envName, enabled)
+}
+
+// CheckCommandAndWarn checks whether an experimental command can be run.
+func CheckCommandAndWarn(cmd *cobra.Command, args []string) error {
```

Comment:
`args` is unused variable.

Reply:
make it `_` so user can use it via cobra command pre-run handler like
```golang
func xxxCmd() *cobra. Command {
...
		PreRunE: experimental.CheckCommandAndWarn,
...
```